content/en/docs/virtualization.md

```diff
@@ -334,3 +334,65 @@ windows.2k22 | Microsoft Windows Server 2022
 windows.2k22.virtio | Microsoft Windows Server 2022 (virtio)
 windows.2k25 | Microsoft Windows Server 2025
 windows.2k25.virtio | Microsoft Windows Server 2025 (virtio)
+
+
+## Virtual Routers
+
+Starting with version [v0.27.0](https://github.com/cozystack/cozystack/releases/tag/v0.27.0),
+Cozystack can deploy virtual routers (also known as "router appliances" or "middlebox appliances").
+This feature allows you to create a virtual router based on a virtual machine instance.
+The virtual router can route traffic between different networks.
+
+## Creating a Virtual Router
+
+Creating a virtual router requires a Cozystack administrator account.
+
+1.  **Create a VM Instance**<br/>
+    Use the standard `vm-instance` and `virtual-machine` packages to create a virtual machine instance.
+    
+1.  **Disable Anti-Spoofing Protection**<br/>
+    Run the following command to disable anti-spoofing protection for the VM instance:
+
+    ```bash
+    kubectl patch virtualmachines.kubevirt.io virtual-machine-example --type=merge \
+        -p '{"spec":{"template":{"metadata":{"annotations":{"ovn.kubernetes.io/port_security": "false"}}}}}'
+    ```
+
+1.  **Restart the Virtual Machine**
+
+    ```bash
+    virtctl stop virtual-machine-example
+    virtctl start virtual-machine-example
+    ```
+
+1.  **Retrieve the IP Address of the VM**
+
+    ```bash
+    kubectl get vmi
+    ```
+
+    The output will have a line with the new VM's IP address:
+
+    ```console
+    NAME                      AGE     PHASE     IP            NODENAME        READY
+    virtual-machine-example   3d4h    Running   10.244.8.56   gld-csxhk-003   True
+    ```
+
+1.  **Configure Custom Routes**<br/>
+    Edit the tenant namespace:
```

Comment:
It's the tenant to whom the new virtual router belongs, isn't it? I'd like to mention it explicitly.

Reply:
Yes